app/assets/javascripts/Components/graders_manager.jsx

```diff
@@ -255,18 +255,21 @@ class GradersManager extends React.Component {
                   <label>
                     <input
                       type="checkbox"
-                      checked={this.state.skip_empty_submissions}
+                      checked={!this.state.skip_empty_submissions}
                       onChange={this.toggleSkipEmptySubmissions}
                       style={{marginRight: "5px"}}
                     />
                     {I18n.t("graders.skip_empty_submissions")}
                   </label>
+                  <div className="inline-help">
+                    <p>{I18n.t("graders.skip_empty_submissions_tooltip")}</p>
+                  </div>
                 </div>
                 <div style={{marginBottom: "1em"}}>
                   <label>
                     <input
                       type="checkbox"
-                      checked={this.state.anonymize_groups}
+                      checked={!this.state.anonymize_groups}
```

Comment:
This label wasn't logically negated ("Do not show" was changed to "Hide"), so the `checked` value shouldn't be negated either.